vision_agent/tools/tools.py

```diff
@@ -1746,6 +1746,107 @@ def closest_box_distance(
     return cast(float, np.sqrt(horizontal_distance**2 + vertical_distance**2))
 
 
+def flux_image_inpainting(
+    prompt: str,
+    image: np.ndarray,
+    mask_image: np.ndarray,
```

Comment:
I'd call this `mask` instead of `mask_image` so it can be easily matched with the `mask` key output from segmentation models